### PR TITLE
added support for ssl certificate chain file

### PR DESCRIPTION
**Pull request by aqulu**
Chrome and Internet Explorer throw ERR_INSECURE_RESPONSE for ssl certificates that require a certificate chain to be trusted

### CODE REVIEW

**Comment by meijdenmedia** [comment]
Nice work!

File: src/cli/cli.ts [hunk 1]
```diff
@@ -91,6 +91,12 @@ export class Cli {
                     return options.protocol == 'https';
                 }
             }, {
+                name: 'sslCertChainPath',
```
Comment (tlaverdure) [inline src/cli/cli.ts]: Since this is not a common configuration, let's omit this from the cli initial setup.

Thanks

Comment (aqulu) [inline src/cli/cli.ts]: thanks for the quick reaction! removed it with the newest commit

Comment (tlaverdure) [inline src/cli/cli.ts]: Thanks, I should be releasing 1.3.1 this week.

**Pull request merged**